src/StreetNameRegistry.Projections.Legacy/StreetNameListV2/StreetNameListProjectionsV2.cs

```diff
@@ -335,10 +335,10 @@ await context.FindAndUpdateStreetNameListItem(
             When<Envelope<MunicipalityOfficialLanguageWasRemoved>>(DoNothing);
             When<Envelope<MunicipalityWasCorrectedToCurrent>>(DoNothing);
             When<Envelope<MunicipalityWasCorrectedToRetired>>(DoNothing);
-            When<Envelope<MunicipalityWasImported>>(DoNothing);
```

Comment:
Deze handler voor `MunicipalityWasImported` verwijderen was intended?

Reply:
Yes, there was already one where it was handled.